apps/velo/get_file.py

```diff
@@ -20,4 +20,5 @@ def get_file(remote_file_path, local_path, filename, site_user, velo_username, p
         raise
 
 
-get_file(sys.argv[1], sys.argv[2], sys.argv[3], sys.argv[4], sys.argv[5], sys.argv[6])
+get_file(sys.argv[1], sys.argv[2], sys.argv[3],
+         sys.argv[4], sys.argv[5], sys.argv[6])
```

Comment:
Can we stick these into variables with useful names first so we have some kind of documentation as to what all of these args mean?